src/content/api/payment-requests.mdx

```diff
@@ -453,7 +453,7 @@ Payment Activities are created when a Payment Request has been `created`, `paid`
       The canonical value of the Payment Request. Must be less than 100000000 and positive.
     </Property>
     <Property name="barcode" type="string">
-      [Scanned Code](/api/scanned-codes) used to create the Payment Request. Required when `preAuth` is `true`.
+      [Scanned Code](/api/scanned-codes) used to create the Payment Request.
```

Comment:
Change: it is still required for quick pay I believe so I think it would be good to mention that.

Reply:
@daallen303 Have updated this to reference the quick pay flow.
<img width="925" alt="image" src="https://github.com/user-attachments/assets/9c92dfa8-83f8-46b4-85d6-b2a9f7ed1066">